.claude/hooks/pre-commit.sh

```
@@ -0,0 +1,14 @@
+#!/bin/bash
+# 커밋 전 품질 검사: TypeScript 타입 검사 → lint
+# 실패 시 exit 2로 차단
+
+cd "$(git rev-parse --show-toplevel 2>/dev/null || pwd)"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)